src/database/models/entitytype.js

```diff
@@ -9,21 +9,47 @@ module.exports = (sequelize, DataTypes) => {
 				primaryKey: true,
 				type: DataTypes.INTEGER,
 			},
-			value: { type: DataTypes.STRING, allowNull: false },
+			value: { type: DataTypes.STRING, allowNull: false, unique: true },
 			label: { type: DataTypes.STRING, allowNull: false },
-			status: { type: DataTypes.STRING, allowNull: false },
-			type: { type: DataTypes.STRING, allowNull: false },
+			status: { type: DataTypes.STRING, allowNull: false, defaultValue: 'ACTIVE' },
 			created_by: { type: DataTypes.INTEGER, allowNull: false },
 			updated_by: { type: DataTypes.INTEGER, allowNull: false },
 			allow_filtering: {
 				type: DataTypes.BOOLEAN,
 				allowNull: false,
 			},
+			data_type: { type: DataTypes.STRING, allowNull: false },
```

Comment:
If this is being set as allowNull false, then we need to standardize the values it can take. Or is the suggestion to just go for "STRING" for now?

Reply:
Add default Value "STRING"

Reply:
Resolved [here](https://github.com/ELEVATE-Project/mentoring/pull/258/files#diff-b4a89c05413a124cef2b14c7268289a97fd1ea31542977c7e7edab2c75b1d23e).